### DISCUSSION TITLE
Verständnisfrage: HM-CFG-LAN bestehende Devices bei Wechsel von fhem nach rm/home assistant

### DISCUSSION THREAD

**Discussion by michelpfirter** [OpenCCU/OpenCCU#2642] [🇩🇪 Fragen&Antworten] [answered]
Hallo,

Ich bin im Begriff mein längst überfälliges Upgrade im Bereich Home Automation durchzuführen.
Dies beinhaltet auch den Umzug von Homematic Devices, welche via HM-CFG-LAN betrieben werden, von fhem nach rm/home assistant.

Das HM-CFG-LAN konnte ich bereits erfolgreich in rm/home assistant (os) aufsetzen. Allerdings sehe ich keine meiner Devices, nur die 50 virtuellen Tasten (HM-RCV-50). AES ist keines aktiviert und hmId konnte ich nirgends eingeben, falls es letztere überhaupt braucht.

Meine Frage ist, bzw. mein Verständnis welches durchaus falsch sein kann und womöglich leider auch ist, ist das die Devices an das HM-CFG-LAN angelernt und gekoppelt sind und nicht an die Software welche das HM-CFG-LAN anspricht, als nicht an fhem oder eben neu rm/ha. Entsprechend hätte ich in meiner "Wunsch"-Vorstellung erwartet, dass ich alle bestehenden Devices in meinem HM-Netzwerk gelistet sehen würde, ohne weiteres zutun. Dies ist leider nicht der Fall, und somit mein Wunsch wohl auch nicht real 😄 

Könnte mir jemand bestätigen, dass ich tatsächlich beim Umzug von fhem nach rm/home assistant bei gleicher, unveränderter HM-CFG-LAN alle Devices neu anlernen muss.

Oder mir einen Hinweis geben, wieso ich meine Devices ohne neues Anlernen nicht sehe.

Vielen Dank

PS: die Suche ergab selbstredend das bei Wechsel des Adapters, also zb HM-CFG-LAN zu CCU neu angelernt werden muss, was für mich Nachvollziehbar ist. Eine Lösung die ich gefunden habe ist der Weg über ein Backup, aber weder habe ich diese Situation noch ist mir bekannt wie ich via HM-CFG-LAN ein Backup machen kann.

**Comment by jens-maus** [comment, marked as answer]
> Könnte mir jemand bestätigen, dass ich tatsächlich beim Umzug von fhem nach rm/home assistant bei gleicher, unveränderter HM-CFG-LAN alle Devices neu anlernen muss.

Genau so ist es. Generell ist es so, dass wenn man von FHEM zu einer CCU/RaspberryMatic umsteigt man quasi bei 0 starten muss. Es gibt keinen Umstieg, kein FHEM Backup das man in RM einspielen lassen kann oder sonst was. Ergo bleibt nur ein komplett frisches anlernen ALLER Homematic Geräte und am besten schmeisst man dann auch gleich den alten `HM-CFG-LAN` in den Müll und nutzt eine der modernen Homematic RF Interfaces wie `HmIP-RFUSB` oder `RPI-RF-MOD` oder ähnliches.

**Reply by michelpfirter** [reply]
Besten Dank für die prompte Antwort. In meinem Zoo ist Homematic vor allem durch Heizkörperthermostaten, Outdoor Bewegungsmelder und Zum Glück nur einem Unterputz Aktor vertreten.
Aus diesem Grund wird die GPIO des Raspberry Pi5 durch ein Razberry 7 besetzt und der HM-CFG-LAN kann so physisch und örtlich getrennt vom Pi mit rm/ha und z-wave werkeln. Entsprechend wäre dann ein nächstes Upgrade in meinem Fall wohl ein zweites Pi mit RPI-RF-MOD ... vermerkt - danke 😄
